script/jenkins/update-beats.sh

```diff
@@ -24,4 +24,5 @@ cleanup() {
 }
 trap cleanup EXIT
 
+go get -u github.com/kardianos/govendor
```

Comment:
`govendor` was missing on jenkins.
Unfortunately I haven't found a proper way yet to let the tests run in a local jenkins env before creating a PR.